.circleci/docker/Dockerfile

```diff
@@ -36,6 +36,7 @@ ENV CXX "${cxx}"
 
 # Whether to build torch and torch_xla libraries with CXX ABI
 ENV _GLIBCXX_USE_CXX11_ABI "${cxx_abi}"
+ENV CFLAGS "${CFLAGS} -D_GLIBCXX_USE_CXX11_ABI=${cxx_abi}"
```

Comment:
It's cleaner to set this explictly in Docker env -- instead of implicitly overwritting in `torch_xla` setup/install.